lib/project_types/theme/commands/push.rb

```diff
@@ -25,13 +25,19 @@ def call(args, _name)
           Themekit.ensure_themekit_installed(@ctx)
         end
 
-        CLI::UI::Frame.open(@ctx.message("theme.push.#{action}ing")) do
+        CLI::UI::Frame.open(@ctx.message("theme.push.#{action}", @ctx.root)) do
+          if remove
+            unless CLI::UI::Prompt.confirm(@ctx.message('theme.push.remove_confirm'))
+              @ctx.abort(@ctx.message('theme.push.remove_abort'))
+            end
+          end
+
           unless Themekit.push(@ctx, files: args, flags: flags, remove: remove)
-            @ctx.abort(@ctx.message("theme.push.error.#{action}ing_error"))
+            @ctx.abort(@ctx.message("theme.push.error.#{action}_error"))
```

Comment:
This is better but I'd still vote for not interpolating variables into keys at all - let's say we're renaming `remove` to `delete` here. Searching for the key `theme.push.error.remove_error` wouldn't find this instance and we'd be left with a hanging untranslated key.

Reply:
The keys are searchable in `messages.rb`, would that be enough for translating?

Reply:
It would be enough for translating, but problematic for the (somewhat rare) case where you're changing the message key itself, since you'd have to look for places where it's used and this one wouldn't show up since the message key isn't searchable for in this instance.